data/mods/gen5/random-teams.ts

```diff
@@ -250,7 +252,7 @@ export class RandomGen5Teams extends RandomGen6Teams {
 		case 'Immunity':
 			return hasAbility['Toxic Boost'];
 		case 'Intimidate':
-			return hasMove['rest'];
+			return (hasMove['rest'] || species.id === 'staraptor');
```

Comment:
```suggestion
			return hasMove['rest'] || species.id === 'staraptor';
```
(you don't need to wrap the entire `return`ed value in parentheses)